registrations/serializers.py

```diff
@@ -125,8 +143,6 @@ def create(self, validated_data):
 
             return signup
 
-        raise DRFPermissionDenied(_("The waiting list is already full"))
```

Comment:
Referring to the comment above, I think this was a better place to raise the exception in terms of readability.

Reply:
Changed back to how it was.